app/Models/Sprint.php

```diff
@@ -22,7 +22,9 @@ public function project()
 	 */
 	public function sprintSnapshots()
 	{
-		return $this->hasMany('SprintSnapshot')->orderBy('created_at', 'desc');
+		return $this->hasMany('SprintSnapshot')
+		            ->select(['id', 'sprint_id', 'created_at', 'total_points'])
+		            ->orderBy('created_at', 'desc');
```

Comment:
Makes totally sense to not load the actual data here. Should maybe give a hint in the method comment that this will return only parts of the SprintSnapshot objects.